### PR TITLE
Inline resize

### PR DESCRIPTION
**Pull request by dmitry-mukhin**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+node_modules/
```
Comment (dmitry-mukhin) [inline .gitignore]: (

File: package.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "uploadcare",
+  "version": "0.0.1",
+  "description": "uploadcare ckEditor plugin",
```
Comment (dmitry-mukhin) [inline package.json]: CKEditor

File: package.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "uploadcare",
+  "version": "0.0.1",
```
Comment (dmitry-mukhin) [inline package.json]: the latest version is 2.1.1

File: styles/plugin.css [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+  cursor: ne-resize;
+}
+
+.bottom-left {
+  cursor: sw-resize;
+}
+
+.bottom-right {
+  cursor: se-resize;
+}
```
Comment (dmitry-mukhin) [inline styles/plugin.css]: (

File: src/tools/search-selected-element.js [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+      return widget.element;
+    }
+  }
+
+  var range = sel.getRanges()[0];
+  if (range) {
+    range.shrink(CKEDITOR.SHRINK_TEXT);
+    return editor.elementPath(range.getCommonAncestor()).contains(needle, 1);
+  }
+}
```
Comment (dmitry-mukhin) [inline src/tools/search-selected-element.js]: (

File: src/plugin.js [hunk 6]
```diff
@@ -11,7 +12,7 @@ CKEDITOR.plugins.add('uploadcare', {
     editInst.editor = editor;
     var getBody = require('./tools/get-body');
     
-    editor.addContentsCss( this.path + 'styles/plugin.css' );
+    CKEDITOR.addCss(require('./styles/resize'));
```
Comment (Zmoki) [inline src/plugin.js]: Why css like string in js file? You use browserify for bundle. You also can use https://www.npmjs.com/package/browserify-css for require css file instead js file.

**Comment by Zmoki** [comment]
@ZNick1982 can you use theme style for buttons?

**Comment by Zmoki** [comment]
@ZNick1982 also need to add title and accessability.

For example, bold button from CKEditor

```html
<a id="cke_10" class="cke_button cke_button__bold cke_button_off" href="javascript:void('Bold')" title="Bold"
   tabindex="-1" hidefocus="true" role="button" aria-labelledby="cke_10_label" aria-haspopup="false"
   onkeydown="return CKEDITOR.tools.callFunction(2,event);" onfocus="return CKEDITOR.tools.callFunction(3,event);"
   onclick="CKEDITOR.tools.callFunction(4,this);return false;">
    <span class="cke_button_icon cke_button__bold_icon"
          style="background-image:url('http://localhost:3004/ckeditor/plugins/icons_hidpi.png?t=G6DE');background-position:0 -24px;background-size:16px;">
        &nbsp;
    </span>
    <span id="cke_10_label" class="cke_button_label cke_button__bold_label" aria-hidden="false">
        Bold
    </span>
</a>
```

**Comment by Zmoki** [comment]
@ZNick1982 bug: after select and delete image buttons stay in editor. I removed them with  backspace. After remove and adding new image, buttons not showing.

**Pull request closed without merging**